partner-portal/src/API/Controllers/ConfigController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+
+namespace Rsbc.Dmf.PartnerPortal.Api.Controllers
+{
+    /// <summary>
+    /// Configuration endpoint
+    /// </summary>
+    [ApiController]
+    [Route("api/[controller]")]
+    [AllowAnonymous]
+    public class ConfigController : Controller
+    {
+        private readonly ILogger<ConfigController> _logger;
+        private readonly IHostEnvironment env;
+        private readonly IConfiguration configuration;
+
+        public ConfigController(ILogger<ConfigController> logger, IHostEnvironment env, IConfiguration configuration)
+        {
+            _logger = logger;
+            this.env = env;
+            this.configuration = configuration;
+        }
+
+        /// <summary>
+        /// Get the client configuration for this environment
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [AllowAnonymous]
+        [ProducesResponseType(typeof(Configuration), 200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(500)]
+        public ActionResult<Configuration> Get()
+        {
+            var config = new Configuration
+            {
+                Environment = env.EnvironmentName,
+                OidcConfiguration = configuration.GetSection("auth:oidc").Get<OidcOptions>()
```

Comment:
this won't return anything, better to just delete this method. In fact, better to delete his entire controller, we don't need it yet and when we do we won't be binding configuration sections to model because George wants the configuration names to have upper case snake case, which wouldn't work with model binding

Reply:
We need this to get environments at the login service level which we add later

Reply:
Deleted the config controller but i think we require this when the login service is implemented.

Reply:
For now i have deleted the config controller. We can add that back once figure out the way to pass the keyclock config